Gemfile

```diff
@@ -2,6 +2,7 @@ source 'https://rubygems.org'
 
 gemspec
 
+gem 'attributor', path: '../attributor'
```

Comment:
need to change that to the real attributor (i.e., let's release a new attributor gem and do it)